test/autorest/complexgroup/zz_generated_primitive.go

```diff
@@ -219,13 +219,13 @@ func (client *PrimitiveClient) GetDateTimeRFC1123(ctx context.Context, options *
 		return Datetimerfc1123WrapperResponse{}, err
 	}
 	if !resp.HasStatusCode(http.StatusOK) {
-		return Datetimerfc1123WrapperResponse{}, client.getDateTimeRfc1123HandleError(resp)
+		return Datetimerfc1123WrapperResponse{}, client.getDateTimeRFC1123HandleError(resp)
```

Comment:
"rfc" in response envelope name needs to be fixed.

Reply:
This is an artifact of the [authoring](https://github.com/Azure/autorest.testserver/blob/master/swagger/body-complex.json#L1642).  The same codegen exists in track 1.  We might be able to get clever and fix it.

Reply:
I dont think rfc are letters that are too commonly placed together in that order, so yeah I agree there could be a "safe" way to fix this naming.